src/transmogrify/dexterity/testing.py

```diff
@@ -150,7 +135,7 @@ def __init__(self, transmogrifier, name, options, previous):
                             creators=["user1"],
                             contributors=["user2"],
                             language="en",
-                            effective="2020-07-10 10:24:00.000000 UTC",
+                            effective="2020-07-10 10:24:00 UTC",
```

Comment:
I probably took this from an actual migration. So it would be nice to keep it that way. See the comment in the test file.

Reply:
Reverted